backend/app/Services/RatingService.py

```diff
@@ -0,0 +1,23 @@
+from typing import List, Optional
+from requests import Session
+from sqlalchemy.orm import Session
+from Models import RatingModel
+from Repositories import RatingRepository
+
+
+class RatingService:
+    @staticmethod
+    def get_all_aspect_ratings(db: Session) -> List[RatingModel]:
+        all_ratings = RatingRepository.get_all_aspect_ratings(db=db)
+        return all_ratings
+
+    @staticmethod
+    def get_rating_by_id(db: Session, id: int) -> Optional[RatingModel]:
```

Comment:
Dit is waar de test op faalt but it looks good so i dont understand

Reply:
Kwam door een circular import, de service die deze service aan riep riep de module aka file aan ipv de class. de methode bestaat echter niet op de module, alleen op de class.